api/src/routes/challenge.test.ts

```diff
@@ -36,6 +36,69 @@ const backendChallengeBody2 = {
   id: backendChallengeId2
 };
 
+// /modern-challenge-completed
+const HtmlChallengeId = '5dc174fcf86c76b9248c6eb2';
+const JsProjectId = '56533eb9ac21ba0edf2244e2';
+const multiFileCertProjectId = 'bd7158d8c242eddfaeb5bd13';
+
+const HtmlChallengeBody = {
+  challengeType: 0,
+  id: HtmlChallengeId
+};
+const JsProjectBody = {
+  challengeType: 5,
+  id: JsProjectId,
+  files: [
+    {
+      contents: 'console.log("Hello There!")',
+      key: 'scriptjs',
+      ext: 'js',
+      name: 'script',
+      history: ['script.js']
+    }
+  ]
+};
```

Comment:
```suggestion
} as const;
```
TS is only complaining below because in principle this array could be emptied. Now it can't.